src/runtime/vm/executable.cc

```diff
@@ -168,7 +168,8 @@ std::string Executable::Stats() const {
 
 TVMByteArray Executable::Save() {
   // Initialize the stream object.
-  strm_ = new dmlc::MemoryStringStream(&code_);
+  dmlc::MemoryStringStream strm(&code_);
+  strm_ = &strm;
```

Comment:
dangling pointer is sort of dangerous. could you just pass the ref to `SaveGlobalSection` etc.

Reply:
OK. I will pass them around.